recipes-bsp/bootfiles/bcm2835-bootfiles.bb

```diff
@@ -7,6 +7,15 @@ inherit deploy
 
 include recipes-bsp/common/firmware.inc
 
+INHIBIT_DEFAULT_DEPS = "1"
+do_package[noexec] = "1"
+do_package_qa[noexec] = "1"
+do_packagedata[noexec] = "1"
+do_package_write_ipk[noexec] = "1"
+do_package_write_rpm[noexec] = "1"
+do_package_write_deb[noexec] = "1"
+do_populate_sysroot[noexec] = "1"
```

Comment:
can we see if ```deltask``` is the right option here ?

Reply:
Reading https://www.yoctoproject.org/docs/2.4.2/bitbake-user-manual/bitbake-user-manual.html#deleting-a-task, I'd say `noexec` is the safer option as it ensures we don't accidentally lose any dependency tasks.

Reply:
yes however, deleting means they dont show up in queue at all which is even better from build point of view, so yes asjust the dependencies as required and delete the tasks

Reply:
Fixed.